### DISCUSSION TITLE
Setting for not remembering project state

### DISCUSSION THREAD

**Discussion by colin-rocks** [zed-industries/zed#23939] [Feature Requests]
### Check for existing issues

- [x] Completed

### Describe the feature

I've seen comments, but no formal issues about this topic.

When I open Zed, it restores project state and open tabs. I want a setting that disables that, and gives me a completely clean slate.

**Comment by JosephTLyons** [comment]
@colin-rocks, does this setting work for you?

```json
"restore_on_startup": "none"
```

**Comment by colin-rocks** [comment]
> @colin-rocks, does this setting work for you?
> 
> ```json
> "restore_on_startup": "none"
> ```

No, that seems to only apply to opening Zed with no args.


To clarify, when I open Zed, I `cd some/working/directoy; zed .`

**Comment by colin-rocks** [comment]
Any chance this will be fixed soon?
